code/datums/materials/meat.dm

```diff
@@ -155,8 +170,9 @@
 
 	if(source.exotic_bloodtype)
 		var/datum/blood_type/blood_type = get_blood_type(source.exotic_bloodtype)
-		color = blood_type.get_color()
-		blood_dna = list("[blood_type.dna_string]" = blood_type)
+		if (blood_type)
+			color = blood_type.get_color()
+			blood_dna = list("[blood_type.dna_string]" = blood_type)
```

Comment:
Should we have a default color for in blood typed meat? Or does the sprite come pre colored

Reply:
It does, I should probably actually change this to use a color transition matrix and avoid changing the color if its default blood red